web/html/src/manager/maintenance/maintenance-windows-edit.js

```diff
@@ -0,0 +1,287 @@
+                            <Button className="btn-default" text="Add Calendar" icon="fa-chevron-down "
+                                    handler={() => this.setState({calendarAdded: !this.state.calendarAdded})} />
+                            :
+                            <div className="panel panel-default">
+                                <div className="panel-heading" style={{padding: "0px"}}>
+                                    <Button text="Add Calendar" icon="fa-chevron-up"
+                                            handler={() => this.setState({calendarAdded: !this.state.calendarAdded})} />
+                                </div>
+                                <div className="panel-body">
+                                    <div className="form-horizontal">
+                                        <div className="form-group">
+                                            <label className="col-md-3 control-label">{t("Calendar")}:</label>
+                                            <div className="col-md-7">
+                                                <Combobox id="calendarSelect" name="calendarSelect"
+                                                          data={this.props.calendarNames}
+                                                          selectedId={this.state.selectedCalendar}
+                                                          onSelect={this.onSelectCalendar}
+                                                />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        }
+                    </div>
+                </div>
+            </Form>
+        )
+    }
+
+    renderCalendarEdit() {
+        const model = {
+            calendarName: this.state.calendarName,
+            strategy: this.state.strategy
+        }
+        return (
+            <Form onChange={model => this.onFormChanged(model)} model={model}>
+                <Text name="calendarName" required type="text" label={t("Calendar Name")}
+                      labelClass="col-md-3" divClass="col-md-6" disabled={this.isEdit()}/>
+                {(this.isEdit() && !this.state.calendarUrl) &&
+                <Check name="strategy" label={<b>{t("Cancel affected actions")}</b>} divClass="col-md-6 col-md-offset-3" />
+                }
+                <div className="form-horizontal">
+                    <div className="form-group">
+                        <label className="col-md-3 control-label">{t("Calendar data")}:</label>
+                        {(!this.isEdit() || this.state.calendarUrl) &&
+                        <div className={"align-center col-md-" + (this.isEdit() ? "5" : "4")} >
+                            <input type="text" className="form-control"
+                                   style={{textOverflow: "ellipsis"}}
```

Comment:
Nothing really dangerous but I am against `inline style`. Much better if you use a `className` to maybe just to the one line CSS rule. If it does not exist you can just create a new one in `susemanager-theme.less` maybe.

Reply:
It does definitely not exists int the `.less` file. But this only seems to be used here.
So I would move everything that's unique to this file to it's own css file and the stuff that might
be reusable to `susemanager-theme.less`